selectel/mks.go

```diff
@@ -154,6 +155,9 @@ func mksClusterV1KubeVersionDiffSuppressFunc(k, old, new string, d *schema.Resou
 		return false
 	}
 	desiredPatch, err := kubeVersionToPatch(new)
+	if desiredPatch == 0 && desiredMinor == currentMinor {
```

Comment:
I think we don't need to check `desiredMinor == currentMinor` here, because we've already checked cases when `desiredMinor > currentMinor` and `desiredMinor < currentMinor` earlier.
On the other hand, if we have an error in the result of  `kubeVersionToPatch(new)`, desiredPatch int value will be always equal to zero, so we don't need to check it too.
To simplify this case we can just return `true` in case of `err != nil` below.

Reply:
Yep, it make sense to simplify this check.